virtool/startup.py

```diff
@@ -140,13 +141,10 @@ async def startup_db(app: App):
     """
     logger.info("Connecting to MongoDB")
 
-    dispatcher_interface = DispatcherClient(app["redis"])
-    await get_scheduler_from_app(app).spawn(dispatcher_interface.run())
-
-    app["db"] = await virtool.db.mongo.connect(
-        app["config"], dispatcher_interface.enqueue_change
+    mongo_database = await virtool.db.mongo.connect(
+        app["config"].db_connection_string, app["config"].db_name
     )
-    app["dispatcher_interface"] = dispatcher_interface
+    return mongo_database
```

Comment:
Get rid of `startup_db()` and call `connect()` directly from `startup_databases`.